content/blog/actions-2-0-0/index.Rmd

```diff
@@ -0,0 +1,155 @@
+---
+output: hugodown::hugo_document
+
+slug: actions-2-0-0
+title: GitHub Actions for R developers, v2
+date: 2022-02-14
+author: Gábor Csárdi
+description: >
+    We have updated our GitHub Actions at `r-lib/actions`. Consider upgrading
+    to the new `v2` version, for faster and more reliable GHA jobs.
+
+photo:
+  url: https://www.pexels.com/photo/industry-technology-power-clock-4069389
+  author: Ronaldo Galeano
+
+# one of: "deep-dive", "learn", "package", "programming", "roundup", or "other"
+categories: [programming] 
+tags: ["GitHub Actions", "r-lib", "actions", "CI", "contionuous integration"]
```

Comment:
I'd reduce this list a bit since it's only used to cross-link other blog posts. Maybe just actions or GitHub actions?